src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -31,7 +31,8 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
             builder.UseMiddleware<FunctionInvocationMiddleware>();
             builder.UseMiddleware<HostWarmupMiddleware>();
 
-            builder.UseWhen(context => !context.Request.Path.StartsWithSegments("/admin"), config =>
+            // /synctriggers also needs specialization to complete. Or we end up with empty triggers payload and no errors.
```

Comment:
Instead of this, just put the `[RequiresRunningHost]` attribute on the synctriggers controller method. See FunctionsController.Invoke for an example.

Reply:
Did not know we had this already. Fixed.